Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -32,7 +32,6 @@
 #import "WKError.h"
 #import "WebAuthenticationRequestData.h"
 #import "WebPageProxy.h"
-#import <AuthenticationServices/ASCOSEConstants.h>
```

Comment:
Was this change intentional? It's not really explained in the changelog (though it looks fine).